tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -278,7 +278,7 @@ describe('amo/components/AddonReviewList', () => {
         .toHaveText(fakeReview.title);
 
       expect(root.find('.AddonReviewList-li p'))
-        .toHaveText(fakeReview.body);
+        .toHaveHTML(`<p>${fakeReview.body}</p>`);
```

Comment:
`toHaveText` does not renders the `dangerouslySetInnerHTML`, it gives blank `p` instead.